scipy/stats/tests/test_distributions.py

```diff
@@ -243,28 +243,28 @@ def setup_method(self):
     def test_rvs_with_mode_shift(self):
```

Comment:
All the tests in this file run in 6 seconds on a single core locally, but consistently take hundreds of seconds on Azure

[Example](https://dev.azure.com/scipy-org/df8ffb44-7275-43bc-90f9-aa5c93ef5c51/_apis/build/builds/5428/logs/71)
```
2020-03-30T21:31:59.0182820Z ========================== slowest 10 test durations ==========================
2020-03-30T21:31:59.0208149Z 417.16s call     sparse/linalg/tests/test_matfuncs.py::TestExpM::test_pascal
2020-03-30T21:31:59.0209799Z 158.83s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_p_zero
2020-03-30T21:31:59.0210801Z 140.96s call     sparse/linalg/eigen/lobpcg/tests/test_lobpcg.py::test_diagonal_data_types
2020-03-30T21:31:59.0212296Z 114.32s call     optimize/tests/test__differential_evolution.py::TestDifferentialEvolutionSolver::test_L8
2020-03-30T21:31:59.0213237Z 73.03s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_new_method
2020-03-30T21:31:59.0214063Z 56.00s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_without_mode_shift
2020-03-30T21:31:59.0215117Z 54.96s call     stats/tests/test_continuous_basic.py::test_cont_basic[kstwo-arg59]
2020-03-30T21:31:59.0216334Z 48.61s call     stats/tests/test_stats.py::TestMGCStat::test_twosamp
2020-03-30T21:31:59.0217461Z 43.14s call     special/tests/test_cdflib.py::TestCDFlib::test_chndtrix
2020-03-30T21:31:59.0218670Z 42.71s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_with_mode_shift
2020-03-30T21:31:59.0573170Z =========================== short test summary info ===========================

(second run)
2020-03-30T21:28:21.5475170Z ========================== slowest 10 test durations ==========================
2020-03-30T21:28:21.5496062Z 193.50s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_p_zero
2020-03-30T21:28:21.5497230Z 145.19s call     sparse/linalg/tests/test_matfuncs.py::TestExpM::test_pascal
2020-03-30T21:28:21.5499560Z 137.34s call     sparse/linalg/eigen/lobpcg/tests/test_lobpcg.py::test_diagonal_data_types
2020-03-30T21:28:21.5500632Z 129.13s call     optimize/tests/test__differential_evolution.py::TestDifferentialEvolutionSolver::test_L8
2020-03-30T21:28:21.5501474Z 97.01s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_new_method
2020-03-30T21:28:21.5504064Z 88.09s call     sparse/linalg/tests/test_expm_multiply.py::TestExpmActionInterval::test_expm_multiply_interval_matrix
2020-03-30T21:28:21.5506913Z 72.80s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_without_mode_shift
2020-03-30T21:28:21.5508052Z 63.02s call     sparse/linalg/tests/test_expm_multiply.py::TestExpmActionInterval::test_expm_multiply_interval_vector
2020-03-30T21:28:21.5509629Z 55.51s call     stats/tests/test_continuous_basic.py::test_cont_basic[kstwo-arg59]
2020-03-30T21:28:21.5510344Z 51.18s call     special/tests/test_mpmath.py::TestSystematic::test_legenp_complex_3
2020-03-30T21:28:21.5808121Z =========================== short test summary info ===========================
```

Reply:
To compare this 

All the tests in this file run in 6 seconds on a single core locally, but consistently take hundreds of seconds on Azure

(before run)
```
2020-03-30T21:31:59.0182820Z ========================== slowest 10 test durations ==========================
2020-03-30T21:31:59.0209799Z 158.83s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_p_zero
2020-03-30T21:31:59.0213237Z 73.03s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_new_method
2020-03-30T21:31:59.0214063Z 56.00s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_without_mode_shift
2020-03-30T21:31:59.0218670Z 42.71s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_with_mode_shift
2020-03-30T21:31:59.0573170Z =========================== short test summary info ===========================

(second before run)
2020-03-30T21:28:21.5475170Z ========================== slowest 10 test durations ==========================
2020-03-30T21:28:21.5496062Z 193.50s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_p_zero
2020-03-30T21:28:21.5501474Z 97.01s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_new_method
2020-03-30T21:28:21.5506913Z 72.80s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_without_mode_shift
2020-03-30T21:28:21.5808121Z =========================== short test summary info ===========================
```

after ~25% faster
```
2020-03-31T00:09:17.2524729Z ========================== slowest 10 test durations ==========================
2020-03-31T00:09:17.2549223Z 132.32s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_p_zero
2020-03-31T00:09:17.2554084Z 54.85s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_new_method
2020-03-31T00:09:17.2558442Z 44.09s call     stats/tests/test_distributions.py::TestGenInvGauss::test_rvs_without_mode_shift
2020-03-31T00:09:17.2912044Z =========================== short test summary info ===========================
```